docs/terminal/curtin.md

```diff
@@ -0,0 +1,49 @@
+---
+  title: Curtin TCU
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Curtin TCU   | Curtin Approach   | 121.000        | CIN_APP                                   |
+
+## Airspace
+
+The limits of the CIN TCU are `SFC` to `F200` within 25 DME CIN. This may be amended by NOTAM.
+
+By default, CIN APP owns the following Restricted Areas, as detailed below:
+
+- R801A (Partial) (`SFC`-`A035`)  
+- R802A (`A035`-`F200`)  
+
+The above Restricted Areas are classified as Class C when CIN APP is active.
+
+!!! note
+    See [VATPAC NOTAMs](https://vatpac.org/publications/notam){target=new} for active NOTAMs which may affect military operations.
+
+## Coordination
+### CIN TCU / CIN ADC
+
+'Next' coordination is required from CIN ADC to CIN TCU for all aircraft.
+
+### CIN TCU / TRT(KIY) 
+#### Departures
+Voiceless coordination is in place from CIN TCU to TRT(KIY) for aircraft:  
+Planned at or above `F200`: `Assigned F200`  
+Planned below `F200`: `Assigned the RFL`  
+
+Any aircraft not meeting the above criteria must be prior coordinated to ENR.
+
+!!! example
+    <span class="hotline">**CIN TCU** -> **TRT(KIY)**</span>: "QFA1601, with your concurrence, will be assigned F160, for my separation with JTE654"  
```

Comment:
Also small thing, but in these examples, don't use "TRT(KIY)" formatting, just do "KIY", cheers